packages/admin/admin/src/appHeader/button/AppHeaderButton.styles.tsx

```diff
@@ -1,43 +1,78 @@
-import { ButtonBaseClassKey } from "@mui/material";
-import { Theme } from "@mui/material/styles";
-import { createStyles } from "@mui/styles";
-
-import { AppHeaderButtonProps } from "./AppHeaderButton";
-
-export type AppHeaderButtonClassKey = ButtonBaseClassKey | "inner" | "startIcon" | "endIcon" | "typography";
-
-export const styles = ({ spacing }: Theme) => {
-    return createStyles<AppHeaderButtonClassKey, AppHeaderButtonProps>({
-        root: {
-            height: "100%",
-            borderLeft: "1px solid rgba(255, 255, 255, 0.2)",
-        },
-        inner: {
-            display: "flex",
-            alignItems: "center",
-            justifyContent: "center",
-            height: "100%",
-            minWidth: "var(--header-height)",
-            boxSizing: "border-box",
-            paddingLeft: spacing(4),
-            paddingRight: spacing(4),
-        },
-        disabled: {},
-        focusVisible: {},
-        startIcon: {
-            display: "flex",
-            alignItems: "center",
-            "&:not(:last-child)": {
-                marginRight: spacing(2),
-            },
-        },
-        endIcon: {
-            display: "flex",
-            alignItems: "center",
-            "&:not(:first-child)": {
-                marginLeft: spacing(2),
-            },
-        },
-        typography: {},
-    });
-};
+import { ButtonBase, Typography } from "@mui/material";
+import { css, styled } from "@mui/material/styles";
+
+export type AppHeaderButtonClassKey = "root" | "inner" | "startIcon" | "endIcon" | "typography";
+
+export const Root = styled(ButtonBase, {
```

Comment:
@jamesricky in all other PRs the .styles files are removed, should we also remove it here?

Reply:
I think it's fine to keep the file if there are a lot of styles/slots.